update Comm.Arrangement Tutorial (#5981)
SAPDocuments/Tutorials: tutorials/abap-environment-communication-arrangement/abap-environment-communication-arrangement.md

[ACCORDION-BEGIN [Step 12: ](Check communication scenario via SAP Cloud Platform Cockpit)]
  1. Logon to your [SAP Cloud Platform Cockpit](https://account.hana.ondemand.com).

    ![check scenario](signIn.png)

  2. Select your global account.

    ![check scenario](global.png)

  3. Select Subaccounts.

    ![check scenario](subaccounts.png)

  4. Select your subaccount.

    ![check scenario](subaccount.png)

>   5. Select Service Instances and find your Instance and open it with clicking arrow.

Reviewer: ```suggestion
  5. Select Service Instances and find your Instance and open it by clicking the arrow.
```

    ![check scenario](serviceInstances.png)

  6. Click Actions to create Service Key.

    ![check scenario](actions.png)

>   7. Create Service key with following parameters:

Reviewer: ```suggestion
  7. Create Service key with the following parameters:
```

     }
    ```

    ![check scenario](serviceKey.png)

>   8. Now your new service key is listed under Service Keys. The successful creation of the service key proved the existence of the communication scenario.

Reviewer: ```suggestion
  8. Now, your new service key is listed under Service Keys. The successful creation of the service key proves the existence of the communication scenario.
```
